slurp/src/main/webapp/scripts/script.js

```diff
@@ -11,18 +11,3 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
```

Comment:
Should this file even be kept? It may make more sense to just delete it and add it back when you need it. Having an empty file in production doesn't make much sense. Cleaning up these files may not be fit for this PR, though, and a separate cleanup PR may be warranted. This PR should probably not reference or touch this file in that case.

Reply:
This file is already removed from master. Not sure why it still shows here.